world/physics/RigidBody.cpp

```diff
@@ -0,0 +1,206 @@
+#include "RigidBody.h"
+
+#include "gravity.h"
+#include "math/EigenWrapper.h"
+
+#ifdef USE_PYBIND_TO_COMPILE
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"   // purposely comparing floats
+#include "pybind11/eigen.h"
+#pragma GCC diagnostic pop
+#endif
+
+RigidBody::RigidBody(double mass, const Matrix_3x3& InertiaTensor, const Vector3& init_pos_b_wrt_g_in_g,
+                     const Quaternion& init_g_q_b, const Vector3& init_vel_b_wrt_g_in_b,
+                     const Vector3& init_omega_b_wrt_g_in_b)
+    : mass_(mass), InertiaTensor_(InertiaTensor) {
+    InertiaTensorInverse_ = InertiaTensor.inverse();
+
+    set_pos_b_wrt_g_in_g(init_pos_b_wrt_g_in_g);
+    set_g_q_b(init_g_q_b);
+    set_vel_b_wrt_g_in_b(init_vel_b_wrt_g_in_b);
+    set_omega_b_wrt_g_in_b(init_omega_b_wrt_g_in_b);
+
+    u_ = Vector6::Zero();
+}
+
+Vector3 RigidBody::get_gravity_b() {
```

Comment:
Use from models/gravity.cpp

Reply:
well, we aren't using the rigid body class at all anyways